config/routes.rb

```diff
@@ -2,7 +2,10 @@
   mount_devise_token_auth_for 'User', at: 'auth'
   
   api_version(:module => "api/v1", path: { value: 'api/v1' }, defaults: { format: :json }) do
-    resources :books,  only: %i[index show] 
-  end
+    resources :books,  only: %i[index show]
 
+    resources :users do
```

Comment:
Careful! this is creating also the rest routes for users, for example: /users/show,create,update, etc

Reply:
Done